_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,419 @@
+---
+id: efbfc7
+name: Auto-updating text content can be paused or stopped
+rule_type: atomic
+
+description: |
+  This rule checks that there are mechanisms to pause or stop the auto-updating of text content
+
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:2.2.2: # Pause, Stop, Hide (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+
+input_aspects:
+  - DOM Tree
+
+authors:
+  - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [visible][] [text node][] in the [flat tree][] of an [HTML document][] if:
+
+- the content of the [text node][] changes through an action that was not initiated by the user; and
+- the change happens anytime after the [readiness][document readiness] of the [HTML document][] the [text node][] belongs to is equal to "complete".
+
+## Expectation 1
+
+For the test target a [mechanism][] is provided to pause or stop the updating of the content of the [text node][].
+
+**Note**: If there is more than one [text node][] with auto-updating content, a single [mechanism][] may be used to pause or stop updating all [text nodes][text node].
+
+## Expectation 2
+
+The [mechanism][] to pause or stop the auto-updating of the content of the [text node][] is [visible][], has an [accessible name][] that is not empty ("") nor only [ASCII whitespace][] , and is [included in the accessibility tree][].
+
+## Assumptions
+
+This rule assumes that the auto-updating of the content is not [essential][], which is listed as valid exception to [SC 2.2.2][]. When the auto-updating of content is [essential][] this rule may produce incorrect results.
```

Comment:
The link to SC 2.2.2 should use the full name (Success Criterion 2.2.2: Pause, Stop, Hide).